### DISCUSSION TITLE
[Q] The worksheet view is frozen, and excel prompts that there is a problem with the view when exporting

### DISCUSSION THREAD

**Discussion by yzkno1** [exceljs/exceljs#2374] [Q&A]
## 💬 Questions and Help
The worksheet view is frozen, and excel prompts that there is a problem with the view when exporting
![8b896abc4ff6aa54de21be4efc6cb17](https://user-images.githubusercontent.com/42439916/129832706-e56c4a25-5203-4633-bdeb-fd34a8e577b4.png)

**Comment by Mercer-L** [comment]
I encountered the same problem, have you solved it?
